doc/flame_svg/install.md

```diff
@@ -0,0 +1,27 @@
+# Installation
```

Comment:
It's better to skip this doc and link to pub.dev's instructions instead, or write it in a way that won't get outdated when the version is updated.

Reply:
Right, since installation instructions are basically the same for all Flame packages (or indeed for all Dart packages), then we shouldn't repeat this same information over and over again.

Reply:
I would remove this whole file and link to pub in a new section in the top of the other file.